### PR TITLE
62: Timesheet daily view.

### PR DESCRIPTION
**Pull request by josue2591**
**Steps to test**
- Add a timesheet node with the starter date of the monday of the current week and the finish date of the sunday of the curret week.
- Add some time entry nodes with different dates between the current week.
- Go to `timesheet/daily` and see if you can select the time entries by chosen the week's day.

[Issue link](https://github.com/ManatiCR/manatime/issues/62)

### CODE REVIEW

**Comment by robherba** [comment]
In the description field, the tag `<p>` is displayed.

File: modules/features/manatime_feature_timesheets/includes/timesheet.inc [hunk 1]
```diff
@@ -303,7 +353,7 @@ class Timesheet {
    * @param string $remove_link_row
    *   A link to remove the record.
    */
-  public function addRecord($project, $task, $time_entries, $total_hours_worked, $remove_link_row) {
+  public function addRecord($project, $task, array $time_entries, $total_hours_worked, $remove_link_row) {
```
Comment (kporras07) [inline modules/features/manatime_feature_timesheets/includes/timesheet.inc]: Why?

File: modules/features/manatime_feature_timesheets/plugins/content_types/timesheet_daily_view.inc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    $node = $context[0]->data;
+    $timesheet = new Timesheet($node);
+    if (isset($context[1]->data)) {
+      if (DateTime::createFromFormat('Ymd', $context[1]->data)) {
+        $day = date('Y-m-d H:i:s', strtotime($context[1]->data));
+        $entries = $timesheet->getTimeEntriesPerDay($day);
+      }
+    }
+    else {
+      $current_date = (new DateTime('now 00:00'))->format('Y-m-d H:i:s');
```
Comment (kporras07) [inline modules/features/manatime_feature_timesheets/plugins/content_types/timesheet_daily_view.inc]: "now 00:00" ? What does it do?

File: modules/features/manatime_feature_timesheets/themes/timesheet-daily-view.tpl.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <th><?php print $table_headers['task']; ?></th>
+        <th><?php print $table_headers['description']; ?></th>
+        <th><?php print $table_headers['duration']; ?></th>
+      </tr>
+    </thead>
+    <tbody>
+      <?php if (!empty($time_entries)): ?>
+        <?php foreach ($time_entries as $time_entry): ?>
+          <tr class="time-entry">
+            <th>
```
Comment (kporras07) [inline modules/features/manatime_feature_timesheets/themes/timesheet-daily-view.tpl.php:22]: remove the p tags. Throw stuff directly in th.

File: themes/custom/manatime_theme/manatime_theme.info [hunk 4]
```diff
@@ -29,3 +29,6 @@ stylesheets[print][] = 'css/print.css'
 ; ========================================
 
 modernizr[tests][] = css_boxsizing
+
+; Panels layouts. You can place multiple layouts under the "layouts" folder.
+plugins[panels][layouts] = layouts
```
Comment (kporras07) [inline themes/custom/manatime_theme/manatime_theme.info]: rename to panels-layouts please (don't forget to rename the folder)

**Comment by kporras07** [comment]
Really minor changes. It looks overall good :D

**Comment by josue2591** [comment]
@kporras07 Ready with the changes.

**Comment by kporras07** [comment]
:shipit:

**Pull request merged**